coalib/results/AbsolutePosition.py

```diff
@@ -0,0 +1,55 @@
+from coalib.results.TextPosition import TextPosition
+from coalib.misc.Decorators import enforce_signature
+
+
+class AbsolutePosition(TextPosition):
+
+    @enforce_signature
+    def __init__(self,
+                 text: (tuple, list, None)=None,
+                 position: (int, None)=None):
+        """
+        Creates an AbsolutePosition object that represents the index of a
+        character in a string.
+
+        :param text:     The text containing the character.
+        :param position: Position identifying the index of character
+                         in text.
+        """
+        line = column = None
+        if position and text:
+            line, column = calc_line_col(text, position)
+        self._text = text
+        self._position = position
+        super().__init__(line, column)
+
+    @property
+    def position(self):
+        return self._position
+
+
+def calc_line_col(text, position):
```

Comment:
i am also open for changing the name, something shorter maybe. As far as encapsulating it into an object goes, this function is being used so that i can create a `TextPosition` in the first place :P